File: day14/code.py
#Leetcode Solution

# class Solution:
#     def maxProductDifference(self, nums: List[int]) -> int:
#         nums.sort()
#         return (nums[-1]*nums[-2])-(nums[0]*nums[1])


def maxProductDifference(nums):
    nums.sort()
    return (nums[-1]*nums[-2])-(nums[0]*nums[1])
    
n=int(input("Enter length of list: "))
nums=[]
for i in range(n):
    nums.append(int(input("Enter score: ")))    
result=maxProductDifference(nums)
print(result)

File: day59/code.cpp
// Leetcode Solution

// class Solution {
// public:
//     bool isPowerOfThree(int n) {
//         if (n==1){
//             return true;
//         }
//         else if(n%3!=0){
//             return false;
//         }
//         if (n<4){
//             if (n==3){
//                 return true;
//             }
//             else{
//                 return false;
//             }
//         }
//         int m = n/3;
//         return isPowerOfThree(m);
//     }
// };